__test__/op-connect.test.ts

```diff
@@ -169,6 +170,52 @@ describe("Test OnePasswordConnect CRUD", () => {
         expect(vaults[0].name).toEqual(title);
         expect(vaults[1].name).toEqual(title);
     });
+
+    describe("get vault by title", () => {
+        const title = "awesome_title";
+
+        test("should throw an error if no vaults found", async () => {
+            const op = OnePasswordConnect(testOpts);
```

Comment:
Is there a reason for redefining the op client in each test rather that on the test group level?

Reply:
Agree. I think it worth to initialise it once for all the test cases.
I just followed the current style.
I'm going to move it to the top level.